x-pack/legacy/plugins/siem/public/pages/network/ip_details/utils.ts

```diff
@@ -6,30 +6,51 @@
 
 import { Breadcrumb } from 'ui/chrome';
 
+import { get } from 'lodash/fp';
 import { decodeIpv6 } from '../../../lib/helpers';
-import { getNetworkUrl } from '../../../components/link_to/redirect_to_network';
+import { getNetworkUrl, getIPDetailsUrl } from '../../../components/link_to/redirect_to_network';
 import { networkModel } from '../../../store/network';
 import * as i18n from '../translations';
+import { NetworkRouteType } from '../navigation/types';
+import { NetworkRouteSpyState } from '../../../utils/route/types';
 
 export const type = networkModel.NetworkType.details;
+const TabNameMappedToI18nKey: Record<NetworkRouteType, string> = {
+  [NetworkRouteType.alerts]: i18n.NAVIGATION_ALERTS_TITLE,
+  [NetworkRouteType.anomalies]: i18n.NAVIGATION_ANOMALIES_TITLE,
+  [NetworkRouteType.flows]: i18n.NAVIGATION_FLOWS_TITLE,
+  [NetworkRouteType.dns]: i18n.NAVIGATION_DNS_TITLE,
+  [NetworkRouteType.http]: i18n.NAVIGATION_HTTP_TITLE,
+  [NetworkRouteType.tls]: i18n.NAVIGATION_TLS_TITLE,
+};
 
-export const getBreadcrumbs = (ip: string | undefined, search: string[]): Breadcrumb[] => {
-  const breadcrumbs = [
+export const getBreadcrumbs = (params: NetworkRouteSpyState, search: string[]): Breadcrumb[] => {
+  let breadcrumb = [
     {
       text: i18n.PAGE_TITLE,
       href: `${getNetworkUrl()}${search && search[0] ? search[0] : ''}`,
     },
   ];
+  if (params.detailName != null) {
+    breadcrumb = [
+      ...breadcrumb,
+      {
+        text: decodeIpv6(params.detailName),
+        href: `${getIPDetailsUrl(params.detailName)}${search && search[1] ? search[1] : ''}`,
+      },
+    ];
+  }
+  if (params.tabName != null) {
```

Comment:
Are the changes here in preparation for adding tabs to `IP Details` page? Just curious as I haven't seen any other IP Details changes in this PR.